files/en-us/mdn/guidelines/writing_style_guide/index.html

```diff
@@ -236,7 +236,7 @@ <h4 id="Tabs_and_line_breaks">Tabs and line breaks</h4>
 
 <h4 id="Inline_code_formatting">Inline code formatting</h4>
 
-<p>Use the "Code" button (labeled with two angle brackets "<strong>&lt;&gt;</strong>") to apply inline code-style formatting to function names, variable names, and method names. (This uses the {{HTMLElement("code")}} element). For example: "the <code class="js plain">frenchText()</code> function".</p>
+<p>Use the {{HTMLElement("code")}} tags to mark up function names, variable names, and method names. For example: "the <code>frenchText()</code> function".</p>
```

Comment:
I don't think that's an appropriate hyperlink. (referring to the use of the `HTMLElement`). You just need to say:
```suggestion
<p>Use the &lt;code&gt; tags to mark up function names, variable names, and method names. For example: <code>the &lt;code&gt;frenchText()&lt;/code&gt; function</code>.</p>
```
(man, this would be easier if it was just Markdown)

Reply:
Fair enough. Fixed. Yes, bring on the markdown!